crawl-ref/source/dat/des/branches/spider.des

```diff
@@ -259,6 +259,33 @@ xxx....x.x.x.x....xxx
         xxxxx
 ENDMAP
 
+NAME:    nicolae_spider_entry_lesser_pyramids
+TAGS:    spider_entry transparent patrolling
+KMONS:   Os = pharaoh ant
+KMONS:   S = wolf spider / redback / tarantella / jumping spider
+KFEAT:   O = enter_spider_nest
```

Comment:
This is probably a bit much for spider entrance; 1 pharaoh ant plus some of the non wolf spider guys is plenty.